includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -831,6 +831,8 @@ private function get_section_export_defaults_main()
             [
               "name"      => WCMP_Settings::SETTING_COUNTRY_OF_ORIGIN,
               "label"     => __("Default country of origin", "woocommerce-myparcel"),
+              "type"      => "select",
+              "options"   => (new WC_Countries)->get_countries(),
```

Comment:
```suggestion
              "options"   => (new WC_Countries())->get_countries(),
```
Heeft wat mij betreft (en in [PSR-12)](https://www.php-fig.org/psr/psr-12/#4-classes-properties-and-methods) de voorkeur